src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const total = days * 40;
+  let rentalCost;
+
+  if (days >= 1 && days < 3) {
+    rentalCost = total;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
Instead of checking if `days >= 3` again, you can use an `else if` statement to make the code more efficient and easier to read.